README.md

```diff
@@ -177,3 +177,49 @@ database.
 During the `handle_command(...)` method you will need to fetch `bot_token` based on the received `team_id`.
 
 </details>
+
+## API Component
+
+The [API component](https://github.com/Lightning-AI/LAI-API-Access-UI-Component) provides you with an API which you can use to hit the server directly to generate results.
+
+<details>
+<summary><b><u>How to use the API</u></b></summary>
+<br>
+
+**Step 1:**
+Run the app.
+
+```bash
+python -m lightning run app app.py --cloud
+```
+
+Alternatively you can use the URL of the public app. You can find that one on lightning.ai/muse (Check step 2).
+
+**Step 2:**
+Retrieve the URL from the bottom of the app page.
+
+<img src="https://lightningaidev.wpengine.com/wp-content/uploads/2022/10/Screenshot-2022-10-20-at-11.39.25-PM.png" width="600px">
+
+**Step 3:**
+Generate results.
```

Comment:
can't generate result from API component

Reply:
you mean the code-snippet doesn't work?

Reply:
I mean, the code snippet is just static. We can generate results in the component. The step title can be "To generate image" 😄

Reply:
done!